src/graph/transform/to_bipartite.cc

```diff
@@ -15,8 +15,7 @@
 #include <dgl/runtime/container.h>
 #include <vector>
 #include <tuple>
-// TODO(BarclayII): currently ToBlock depend on IdHashMap<IdType> implementation which
-// only works on CPU.  Should fix later to make it device agnostic.
```

Comment:
Given that we have to_block implemented on the GPU, this comment didn't seem relevant/correct anymore.